hack/cluster-openshift.sh

```diff
@@ -178,12 +178,26 @@ OPENSHIFT_BIN_PATH="${OPENSHIFT_BIN_PATH:=${HOME}/bin}"
 # This is the IP address where OpenShift will bind its master.
 # This should be a valid IP address for the machine where OpenShift is installed.
 # NOTE: Do not use any IP address within the loopback range of 127.0.0.x.
-OPENSHIFT_IP_ADDRESS=${OPENSHIFT_IP_ADDRESS:-`echo $(ip -f inet addr | grep 'state UP' -A1 | tail -n1 | awk '{print $2}' | cut -f1 -d'/')`}
+if which ip > /dev/null ; then
+  OPENSHIFT_IP_ADDRESS=${OPENSHIFT_IP_ADDRESS:-`echo $(ip -f inet addr | grep 'state UP' -A1 | tail -n1 | awk '{print $2}' | cut -f1 -d'/')`}
+else
+  if [ ! "$OPENSHIFT_IP_ADDRESS" ] ; then
+    echo "ERROR: can't detect IP address please specify with --address argument"
```

Comment:
Rather than abort and require the option, just default to 127.0.0.1 - that should be fine under most conditions (and it is usually OK for most people anyway to bind to the loopback address). So I would delete lines 185 and 186 and replace it with the line `OPENSHIFT_IP_ADDRESS="127.0.0.1"`

Reply:
Sure I can do that, but the comment on line 180 says "NOTE: Do not use any IP address within the loopback range of 127.0.0.x."
I'll remove that also if you think it's incorrect

Reply:
I don't know why that is in there... I think I was thinking if you use 127.x then you can't get to your cluster from another machine (which is why I originally added this option to change the IP address :)) I would remove it. I think it will work.